docker-compose/standalone-dashboard/docker-compose.yml

```diff
@@ -23,9 +23,9 @@ services:
 
   standalone:
     image: apachepulsar/pulsar
-    expose:
-      - 8080
-      - 6650 
+    ports:
```

Comment:
can you separate this change into a separated pull request?

Reply:
Ok  will do 
Thanks 
Kishore